style.css

```diff
@@ -416,27 +455,463 @@ ul{
+        align-items: center;
+        padding: 7rem 0;
+        position: absolute;
+        width: 566px;
+        height: 798px;
+        left: 143px;
+        top: 72px;
+    }
+    .primary-text{
+        width: 566px;
+        height: 160px;
+        margin-bottom: 50px;
+    }
+    .supporting-text{
+        width: 566px;
+        height: 24px;
+        font-family: 'Inter';
+        font-style: normal;
+        font-weight: 500;
+        font-size: 32px;
+        line-height: 44px;
+        display: flex;
+        align-items: center;
+    }
+    .description{
+        width: 566px;
+        height: 123px;
+        font-family: 'Inter';
+        font-style: normal;
+        font-weight: 400;
+        font-size: 20px;
+        line-height: 32px;
+        color: #F4F5F7;
+        flex: none;
+        order: 2;
+        flex-grow: 0;
+    }
+    .works-section{
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        position: absolute;
+        width: 100%;
+        height: auto;
+        left: 0;
+        /* top: 875px; */
+        gap: 24px;
+        padding: 0;
+    }
+    .works-header{
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+        justify-content: left;
+        width: 100%;
+        flex: none;
+        order: 0;
+        flex-grow: 0;
+        padding: 130px 150px  130px 150px;
+    }
+    .header-text{
+        width: 100%;
+        height: 52px;
+        font-family: 'Crete Round';
+        font-style: normal;
+        font-weight: 400;
+        font-size: 40px;
+        line-height: 52px;
+        display: flex;
+        align-items: center;
+        padding-top: 0px;
+        white-space: nowrap;
+    }
+    .hl{
+        display: inline;
+        width: 60%;
+        margin-right: 10%;
+        border: 1px solid #212121;
+    }
+    .main-work-section{
+        display: flex;
+        flex-direction: row;
+        gap: 24px;
+        width: 80%;
+        left: 0px;
+        margin: auto 10%;
+    }
+    .img-placeholder{
+        width: 65%;
+        height: auto;
+        flex: none;
+        order: 0;
+        flex-grow: 0;
+        align-self: flex-start;
+    }
+    .work-container{
+        display: flex;
+        flex-direction: column;
+        align-items: flex-start;
+        padding: 0px;
+        gap: 20px;
+        width: 448px;
+        height: 348px;
+        flex: none;
+        order: 1;
+        flex-grow: 0;
+        top: 1643px;
+        margin-right: 10%;
+    }
+    .work-description{
+        width: 70%;
+        height: auto;
+        padding: 0;
+        margin: 0;
+    }
+    .main-btn.work-sec{
+        margin: 0;
+    }
+    .work-cards-container{
+        display: flex;
+        flex-direction: column;
+        justify-content:  center;
+        gap: 24px;
+        width: 100%;
+        height: 1379px;
+        flex: none;
+        order: 1;
+        flex-grow: 0;
+        top: 1643px;
+    }
```

Comment:
- [ ]Good job Implementing the changes requested by the previous reviewer, I think while making the changes there is an issue introduced, now your projects cards section are covering the first featured project please see 
![image](https://user-images.githubusercontent.com/97521557/208243705-f3b02b2f-7e70-4b65-bf2d-61f93167b494.png)

Please fix this layout issue and you will be done :+1:

Reply:
Thank you for your feedback I have changed the position to resolve overlappings.